spec/views/granules/descriptor_document_spec.rb

```diff
@@ -153,22 +153,30 @@
     assign(:client_id_model, stub_client_id)
 
     render
-    expect(rendered).to include("template=\"http://ceocat.ccrs.nrcan.gc.ca/cgi-bin/opensearch_r1.sh?q=&amp;id={geo:id?}&amp;bbox={geo:box?}&amp;dtstart={time:start?}&amp;dtend={time:end?}&amp;pw={startPage?}&amp;startIndex={startIndex?}&amp;count={count?}&amp;clientId=foo\">")
+    expect(rendered).to include("template=\"https://eodms-sgdot.nrcan-rncan.gc.ca/apps/cgi/opensearch/eodms_opensearch_r1.sh?q=&amp;id={geo:id?}&amp;bbox={geo:box?}&amp;dtstart={time:start?}&amp;dtend={time:end?}&amp;pw={startPage?}&amp;startIndex={startIndex?}&amp;count={count?}&amp;clientId=foo\">")
   end
   it "is possible to create a collection-specific granule open search descriptor document with a blank client id" do
     stub_client_id = stub_model(ClientId)
     assign(:client_id_model, stub_client_id)
 
     render
-    expect(rendered).to include("template=\"http://ceocat.ccrs.nrcan.gc.ca/cgi-bin/opensearch_r1.sh?q=&amp;id={geo:id?}&amp;bbox={geo:box?}&amp;dtstart={time:start?}&amp;dtend={time:end?}&amp;pw={startPage?}&amp;startIndex={startIndex?}&amp;count={count?}\">")
+    expect(rendered).to include("template=\"https://eodms-sgdot.nrcan-rncan.gc.ca/apps/cgi/opensearch/eodms_opensearch_r1.sh?q=&amp;id={geo:id?}&amp;bbox={geo:box?}&amp;dtstart={time:start?}&amp;dtend={time:end?}&amp;pw={startPage?}&amp;startIndex={startIndex?}&amp;count={count?}\">")
   end
   it "is possible to create a collection-specific granule open search descriptor document with a whitespace client id" do
     stub_client_id = stub_model(ClientId)
     stub_client_id.clientId = ' '
     assign(:client_id_model, stub_client_id)
 
     render
-    expect(rendered).to include("template=\"http://ceocat.ccrs.nrcan.gc.ca/cgi-bin/opensearch_r1.sh?q=&amp;id={geo:id?}&amp;bbox={geo:box?}&amp;dtstart={time:start?}&amp;dtend={time:end?}&amp;pw={startPage?}&amp;startIndex={startIndex?}&amp;count={count?}\">")
+    expect(rendered).to include("template=\"https://eodms-sgdot.nrcan-rncan.gc.ca/apps/cgi/opensearch/eodms_opensearch_r1.sh?q=&amp;id={geo:id?}&amp;bbox={geo:box?}&amp;dtstart={time:start?}&amp;dtend={time:end?}&amp;pw={startPage?}&amp;startIndex={startIndex?}&amp;count={count?}\">")
+  end
+  it "creates a collection-specific granule open search descriptor document with a temporal range limit of 14 days" do
+    stub_client_id = stub_model(ClientId)
+    stub_client_id.clientId = 'foo'
+    assign(:client_id_model, stub_client_id)
+
+    render
+    expect(rendered).to include("<param:Parameter name=\"dtend\" value=\"{time:end}\" minimum=\"0\"  minInclusive = '' maxInclusive = '' maxPeriod=\"P14D\" relativeTo=\"{time:start}\" title=\"Temporal End\" />")
```

Comment:
nitpick: space before = with minInclusive and maxInclusive.

Reply:
Updated